lib/basedriver/commands/find.js

```diff
@@ -62,6 +68,74 @@ commands.findElementsFromElement = async function (strategy, selector, elementId
   return await this.findElOrElsWithProcessing(strategy, selector, true, elementId);
 };
 
+/**
+ * Find an element using a custom plugin specified by the customFindModule cap.
+ *
+ * @param {string} selector - the selector which the plugin will use to find
+ * elements
+ * @param {boolean} multiple - whether we want one element or multiple
+ *
+ * @returns {WebElement} - WebDriver element or list of elements
+ */
+commands.findByCustom = async function (selector, multiple) {
+  if (!this.opts.customFindModule) {
+    throw new Error("Finding an element using a pluagin is currently an " +
+                    "incubating feature. To use it you must manually install a " +
+                    "plugin module inside your Appium node_modules tree, " +
+                    "and set the capability 'customFindModule' to the name of " +
+                    "that module so that it can be required.");
+  }
+
+  let finder;
+  try {
+    finder = require(this.opts.customFindModule);
+  } catch (err) {
+    throw new Error(`Could not load your custom find module. Original error: ${err}`);
+  }
+
+  if (!_.isFunction(finder.find)) {
```

Comment:
`finder && !_.isFunction ..`. would be safer

Reply:
hmm, i guess it's possible that a module is not truthy?